.mise/tasks/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#MISE description="Build the project"
```

Comment:
This script is very much tailored to building on Linux – that's fine by me, but I would make that explicit by renaming the description and the file name accordingly.